book/03-git-branching/sections/branch-management.asc

```diff
@@ -73,3 +73,42 @@ $ git branch --no-merged master
   featureB
 ----
 ====
+
+==== Changing a branch name
+
+You can change a branch name with the `git branch -m` command.
+If you want to collaborate with others, on the newly changed branch, you'll need to push it.
```

Comment:
Ooh, one last thing: you've got one too many commas in here.

```suggestion
If you want to collaborate with others on the newly changed branch, you'll need to push it.
```

Reply:
Whoops, you're right. It's still a bit hard to figure out where a comma should go, and where it should be omitted... 😕 

I usually err on the side of having too many, than having too few...